### PR TITLE
Delegate `legalaidlearning.justice.gov.uk` to Cloud Platform

### PR DESCRIPTION
**Pull request by AntonyBishop**
## 👀 Purpose

- The PR delegates `legalaidlearning.justice.gov.uk` to Cloud Platform.

## ♻️ What's changed

- Update CNAME `legalaidlearning.justice.gov.uk` to NS Records

## 📝 Notes

- [Request](https://groups.google.com/a/digital.justice.gov.uk/g/domains/c/wDWBA14dZtg/m/EAxXcIejAgAJ)

### CODE REVIEW

**Review by PepperMoJ — APPROVED** [review]

**Comment by AntonyBishop** [comment]
On hold awaiting approval from Service Team to deploy changes.

**Comment by AntonyBishop** [comment]
Service Team approved deployment.

**Pull request merged**
